iotlabsshcli/sshlib/open_a8_ssh.py

```diff
@@ -51,40 +50,62 @@ def _cleanup_result(result):
     return result
 
 
-def _nodes_from_groups(group):
-    """Return the full node list from nodes grouped by sites.
-
-    >>> _nodes_from_groups({'saclay': ['node-a8-1.saclay.iot-lab.info',
-    ...                                'node-a8-2.saclay.iot-lab.info'],
-    ...                     'grenoble': ['node-a8-10.grenoble.iot-lab.info',
-    ...                                  'node-a8-11.grenoble.iot-lab.info']})
-    ['node-a8-10.grenoble.iot-lab.info', 'node-a8-11.grenoble.iot-lab.info', \
-'node-a8-1.saclay.iot-lab.info', 'node-a8-2.saclay.iot-lab.info']
+def _extend_result(result, new_result):
+    """ Extend result dictionnary values with new result
+    dictionnary values
+
+    >>> sorted(_extend_result(
```

Comment:
`_extend_result` could return the result sorted. That would make it more consistent.